core-bundle/tests/Image/Studio/StudioTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Tests\Image\Studio;
+
+use Contao\CoreBundle\Asset\ContaoContext;
+use Contao\CoreBundle\Framework\ContaoFramework;
+use Contao\CoreBundle\Image\ImageFactoryInterface;
+use Contao\CoreBundle\Image\PictureFactoryInterface;
+use Contao\CoreBundle\Image\Studio\FigureBuilder;
+use Contao\CoreBundle\Image\Studio\ImageResult;
+use Contao\CoreBundle\Image\Studio\LightBoxResult;
+use Contao\CoreBundle\Image\Studio\Studio;
+use Contao\CoreBundle\Tests\TestCase;
+use PHPUnit\Framework\MockObject\MockObject;
+use Psr\Container\ContainerInterface;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
+
+class StudioTest extends TestCase
+{
+    public function testCreateFigureBuilder(): void
+    {
+        $studio = $this->getStudio();
+
+        $figureBuilder = $studio->createFigureBuilder();
+        $this->assertInstanceOf(FigureBuilder::class, $figureBuilder);
```

Comment:
<img width="842" alt="" src="https://user-images.githubusercontent.com/1192057/87564141-54fb5780-c6c0-11ea-949e-d6e9a6158379.png">

Reply:
I removed/changed the tests as discussed. 

See 56b7f9e21753077e067ae286cdf5401115271965